lib/syskit/test/profile_assertions.rb

```diff
@@ -465,6 +434,62 @@ def assert_can_deploy_together(*actions)
                       e.message, e.backtrace
             end
 
+            # Tests that the following syskit-generated actions can be ALL deployed at the
+            # same time, that is they result in a valid, non-abstract network whose all
+            # components have a deployment
+            #
+            # When resolving actions that are not directly defined from profile
+            # definitions, the method will attempte to resolve method action by
+            # calling them. If there is a problem, pass the action model to the
+            # `exclude` argument.
+            #
+            # In particular, in the presence of action methods with required
+            # arguments, run one assert first with the action method excluded and
+            # another with that action and sample arguments.
+            #
+            # @param action_or_profile if an action interface or profile, test all
+            #   definitions that are reachable from it. In the case of action interfaces,
+            #   this means looking into method actions and action state machines.
+            # @param together_with test that all actions in `action_or_profile`
+            #   can be instanciated when all actions in `together_with` are instanciated
+            #   at the same time. This can be used if the former depend on the presence
+            #   of the latter, or if you want to test against conflicts.
+            def assert_can_deploy_all(
+                action_or_profile = subject_syskit_model,
+                exclude: [], together_with: []
+            )
+                actions = assert_actions(action_or_profile, exclude: exclude)
+
```

Comment:
Newlines are meant to separate "paragraphs" of code. When you have three lines of text, you don't put newlines between them ...